spotifyPlayer.py

```diff
@@ -74,3 +74,19 @@ def checkPlayStatus(fileName):
 		threading.Thread(target=checkPlayStatus, args=(currFileName,)).start()
 
 		return True
+
+	def pause(self):
+		mixer.music.pause()
+		return True
+
+	def resume(self):
+		mixer.music.unpause()
+		return True
+
+	def volume(self, value):
```

Comment:
This isn't C, don't return `-1` or `value`. In fact, don't return anything. Just use the defaults silently and document this behavior. e.g.:

```python
vol = max(0, min(100, value)) / 100
```